Taskfile.yml

```diff
@@ -46,6 +46,12 @@ tasks:
       - echo "Test..."
       - go test ./...
 
+  # task test:checker -- negative-positive
+  test:checker:
+    deps: [ test:gen ]
```

Comment:
thanks for adding the generation here too, I tried yesterday and I faced misunderstanding :+1: